modules/Makefile.docker-machine

```diff
@@ -0,0 +1,54 @@
+DOCKER_MACHINE_NAME ?= gladly
+DOCKER_DOWNLOAD_URL ?= https://get.docker.com/builds/$(LOCAL_OS)/$(LOCAL_ARCH)/docker-$(DOCKER_VERSION)
+DOCKER_ISO ?= https://github.com/boot2docker/boot2docker/releases/download/v$(DOCKER_VERSION)/boot2docker.iso
+
+DEPS_TARGETS += machine:deps
+
+.PHONY : machine\:deps machine\:info machine\:recreate machine\:create machine\:destroy machine\:start machine\:stop machine\:restart machine\:deps
+
+## Setup dependencies for docker-machine
+machine\:deps:
+	@[ -x $(DOCKER_CMD) ] || (curl -s -o $(DOCKER_CMD) $(DOCKER_DOWNLOAD_URL) && chmod 755 $(DOCKER_CMD))
+	@(docker-machine status $(DOCKER_MACHINE_NAME) 2>&1 | grep -v -q "Host does not exist") || $(SELF) machine:create
+	@(docker-machine status $(DOCKER_MACHINE_NAME) 2>&1 | grep -q "^Running") || $(SELF) machine:start
+	@docker-machine env $(DOCKER_MACHINE_NAME) | tr -d '"' > "$(MAKEFILE_PATH).env"
+	$(eval -include $(MAKEFILE_PATH).env)
+	@$(call assert,DOCKER_HOST)
+	@$(call assert,DOCKER_CERT_PATH)
+
+## Docker Machine Info
+machine\:info:
+	@echo 'DOCKER_MACHINE_NAME=$(DOCKER_MACHINE_NAME)'
+	@echo 'DOCKER_DOWNLOAD_URL=$(DOCKER_DOWNLOAD_URL)'
```

Comment:
I think that this would output a different `DOCKER_DOWNLOAD_URL` than what is on the machine if the machine was previously built with a different `DOCKER_VERSION`.